.chloggen/dockerstatsreceiver-add-windows-support.yaml

```
@@ -0,0 +1,7 @@
+change_type: enhancement
+component: dockerstatsreceiver
+note: "Add Windows support"
+issues: []
+subtext: |
+  The dockerstatsreceiver now supports Windows containers.
```

```suggestion
  The dockerstatsreceiver now supports Windows hosts.
```

In principle it shouldn't matter if the container mode is set for Windows or Linux, correct?